giraffe/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@influxdata/giraffe",
-  "version": "2.13.1",
+  "version": "2.13.2",
```

Comment:
I suggest bumping this 2.14.0 because we're defining something new here rather than fixing a bug.